src/modm/driver/display/hd44780_base_impl.hpp

```diff
@@ -177,6 +177,7 @@ modm::Hd44780Base<DATA, RW, RS, E>::writeCGRAM(uint8_t character, uint8_t *cg)
 	while(not writeCommand(SetCGRAM_Address | (character << 3)))
 		;
 	for (std::size_t ii = 0; ii < 8; ++ii) {
+        modm::delay_ms(10);
```

Comment:
taaaaabs ;-P

Reply:
This an older edit without clang... forgive me ohhhh uC-whisperer